### ISSUE TITLE
Chinese tutorial lacks of maintenance

### ISSUE THREAD

**Issue by HuStmpHrrr** [ocaml/v2.ocaml.org#640]
I went thru Chinese tutorial a few days ago and i found it's quite not complete and lacks of maintenance(fewer articles and incomplete articles).

I am Chinese myself and I would like to contribute to translate the English version to Chinese. How can I do that?

I coded sml/nj before for months and am now learning OCaml and coding both java and c# at work. I've also done lots of C, bash, and perl. Hope i can help.

**Comment by Chris00** [comment]
On Mon, 30 Mar 2015 11:51:45 -0700, HuStmpHrrr wrote:

> I went thru Chinese tutorial a few days ago and i found it's quite not complete and lack of maintenance(fewer articles and incomplete articles).
> 
> I am Chinese myself and I would like to contribute to translate the English version to Chinese. How can I do that?

Thanks for your offer, we really appreciate it.  Basically, for each page x.md, create a corresponding page x.zh.md containing the translation ("zh" is the ISO 639-1 code for Chinese https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).

You have two ways to do that, depending with your familiarity with Git.
- If you know Git, I recommend that you clone the repository, add translation files (the content of the website is under — surprise! — site/) and submit pull requests.
- If you are unfamiliar with Git, Github lets you add and edit files online.

Let me know if I was not clear enough.

> I coded sml/nj before for months and am now learning OCaml and coding both java and c# at work.

Welcome to the community!

**Comment by dsheets** [comment]
Hi! Thanks for your offer to help. :-) The easiest way to get started is to click the pencil icon on the top right of the page that you would like to edit/update/correct. This will take you to the source markdown for the page in the language you were browsing. From there, simply click the pencil icon next to the "History" button on the source file and you will be presented with the GitHub in-browser markdown editor. Make your changes and then click "Propose file change" to submit a PR.

**Comment by XVilka** [comment]
@HuStmpHrrr so, any updates on this?

**Comment by HuStmpHrrr** [comment]
@XVilka yeah, I was looking into OCaml a few years ago intending to learn something new but I dropped it for long already. I indeed translated a few chapters and got them merged back then but I do not have any recent plan to pick up the rest. also as a developer I've passed the stage requiring comprehensive tutorials, so really I myself has much less interest in tutorials in general.

However, I got involved into Coq since a year ago and am considering recently the necessity to write decision procedures in OCaml. Though I am not sure how, say, English version is maintained. How regularly the tuto's are revised?
